GenderPayGap.WebUI/GenderPayGap.WebUI.csproj

```diff
@@ -24,6 +24,7 @@
 
   <ItemGroup>
     <Content Remove="Views\Organisation\ManageOrganisationBreadcrumbs.cshtml" />
+    <Content Remove="wwwroot\styles\pages\_employer-reports-table.scss" />

```

Comment:
We're not doing the same thing with files in that folder as far as I can tell, do you know why this is here?

Reply:
Not a clue @CodrinCotarlanSoftwire?

Reply:
I believe we've added a custom style file with a class to hide the tables on desktop / mobile, and I noticed there was a reference to `_employer_reports_table.scss`. However the file doesn't exist, so I removed it from an imports file.

Reply:
First off, it does exist 😛
![image](https://user-images.githubusercontent.com/30173746/192744645-f869b566-26f3-459b-8f4d-042ac3ea89e7.png)
I presume it's here in the `.csproj` because it's not being used anywhere, so it's automatically being excluded from the build? I think we need to add that import back in or actually delete this file if its style isn't being used.

Reply:
I was thinking on another file that we indeed removed. I don't know why this one is in a 'Remove' field in csproj...

Reply:
Removed